core/includes/form.inc

```diff
@@ -4726,8 +4726,14 @@ function theme_textarea($variables) {
 
   // Add resizable behavior.
   if (!empty($element['#resizable'])) {
-    backdrop_add_library('system', 'backdrop.textarea');
-    $wrapper_attributes['class'][] = 'resizable';
+    $valid_values = array('none', 'both', 'horizontal', 'vertical');
```

Comment:
There should also be a conversion from `$element['#resizable'] === FALSE` to `'none'`.